src/components/Shopping/CartGlassesItem.tsx

```
@@ -0,0 +1,149 @@
+    { id: 'glasses1', count: 1 },
+    { id: 'glasses2', count: 1 }
+  ]);
+
+  const toggleItem = (id: string) => {
+    setCheckedItems(prev => ({
+      ...prev,
+      [id]: !prev[id]
+    }));
+  };
+
+  const updateCount = (id: string, delta: number) => { };
```

ESLint

[error] 'id' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)